api/tests/mocha/middleware/connected-user-log.spec.js

```diff
@@ -0,0 +1,37 @@
+const chai = require('chai');
+const sinon = require('sinon');
+const auth = require('../../../src/auth');
+const middleware = require('../../../src/middleware/connected-user-log');
+const connectedUserLogService = require('../../../src/services/connected-user-log');
+
+let req;
+let res;
+let next;
+
+describe('Connected Users Log middleware', () => {
+  beforeEach(() => {
+    req = {};
+    res = {};
+    next = sinon.stub();
+    sinon.stub(auth, 'getUserCtx');
+    sinon.stub(connectedUserLogService, 'save');
```

Comment:
With the changes above you'll need to change the mock to `resolve` to get this test to work. That's a good thing!